plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/TestSqlServerConnectorTest.java

```diff
@@ -195,4 +220,24 @@ public static Object[][] timestampTypes()
                 {"timestamp(12)"}
         };
     }
+
+    @DataProvider
+    public Object[][] testTableNameDataProvider()
+    {
+        return testTableNameTestData().stream()
+                .collect(toDataProvider());
+    }
+
+    private List<String> testTableNameTestData()
+    {
+        return ImmutableList.<String>builder()
+                .add("tricky")
+                .add("[tricky]")
+                .add("[tri[]cky]")
+                .add("[tri[ ]cky]")
+                .add("[tr; i[ ]cky]")
```

Comment:
in this example the brackets don't seem significant. i.e. i think this is testing `;` instead of brackets.

i.e. a test case's significance should either be obvious or added as a comment.